package.json

```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "scripts": {
     "start": "ng serve",
+    "start:https": "ng serve --ssl true --host 0.0.0.0 --port 8443",
```

Comment:
Can we add a line in README.md to explain how to experience Oauth based application?